File: Codeeval/Scala/Easy/Sum of Integers from a FIle.scala
object Main extends App {
  val source = scala.io.Source.fromFile(args(0))
  val lines = source.getLines.filter(_.length > 0)
  var full = 0
  for (l <- lines) {
    full+= l.toInt
  }
  println(full)
}

//Print out the sum of integers read from a file.

File: Codeeval/Java/Easy/Sum of Integers from a File.java
import java.io.*;
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        int full = 0;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            full+=Integer.parseInt(line);
        }
        System.out.println(full);
    }
}

//Print out the sum of integers read from a file.
